test/app-tap/gh-2717-no-quit-sigint.test.lua

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/env tarantool
+
+local tap = require('tap')
+local popen = require('popen')
+
+--
+-- gh-2717: tarantool console quit on sigint
+--
+
+local TARANTOOL_PATH = arg[-1]
+local prompt = 'tarantool> '
+local test = tap.test('gh-2717')
+
+test:plan(2)
+local ph = popen.shell(TARANTOOL_PATH .. ' -i', 'r')
+assert(ph.pid, 'popen error while executing ' .. TARANTOOL_PATH)
+
+local output = ph:read()
+while output:find('tarantool>') == nil do
+    output = output .. ph:read()
+end
+
+ph:signal(popen.signal.SIGINT)
+
+while output:find('C\ntarantool>') == nil do
+    output = output .. ph:read()
+end
+output = output .. ph:read()
+
+test:like(ph:info().status.state, popen.state.ALIVE, 'SIGINT doesn\'t kill tarantool in interactive mode ')
+test:like(output, prompt .. '^C\n' .. prompt, 'Ctrl+C discards the input and calls the new prompt')
+ph:close()
+
+os.exit(test:check() and 0 or 1)
```

Comment:
Nit: please add a newline at the end of file.

Reply:
Added.